bitnami/influxdb/templates/NOTES.txt

```diff
@@ -0,0 +1,144 @@
+
+** Please be patient while the chart is being deployed **
+
+{{- if eq .Values.architecture "high-availability" }}
+
+InfluxDB can be accessed through following DNS names from within your cluster:
+
+    InfluxDB Relay (write operations): {{ include "influxdb.fullname" . }}-relay.{{ .Release.Namespace }}.svc.cluster.local (port {{ .Values.relay.service.port }})
+    InfluxDB servers (read operations):  {{ include "influxdb.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local (port {{ .Values.influxdb.service.port }})
```

Comment:
I think it's worth making `cluster.local` a parameter, such as `.Values.clusterDomain`.

Reply:
I don't see any benefit on doing so.

Reply:
We have had support tickets in the past due to hardcoding `cluster.local`. Here's an example: https://github.com/helm/charts/issues/13230#issuecomment-493455435